doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,128 @@
+*********************
+
+=============  ===============================================================
+Labels         ``priority:{high|medium|low}``
+Applicable to  Issues only
+Description    To classify the priority/importance of a bug or feature
+=============  ===============================================================
+
+Note: Issue priorities are decided/changed during the bug-triage or TSC
+meetings.
+
+Miscellaneous labels
+********************
+
+For both PRs and issues
+=======================
+
++-------------------+----------------------------------------------------------+
+|``Bug``            | The issue is a bug, or the PR is fixing a bug            |
++-------------------+----------------------------------------------------------+
+|``Coverity``       |A Coverity detected issue or its fix                      |
++-------------------+----------------------------------------------------------+
+|Waiting for        | The Zephyr developers are waiting for the submitter to   |
+|author response    | respond to a question, or address an issue.              |
++-------------------+----------------------------------------------------------+
+|``RFC``            |The author would like input from the community. For a PR  |
+|                   |it should be considered a draft                           |
++-------------------+----------------------------------------------------------+
+|``LTS``            |Long term release branch related                          |
++-------------------+----------------------------------------------------------+
+|``In progress``    |For PRs: is work in progress and should not be merged yet.|
+|                   |For issues: Is being worked on                            |
++-------------------+----------------------------------------------------------+
+|``EXT``            | Related to an external component (in ``ext/``)           |
++-------------------+----------------------------------------------------------+
+
+PR only labels
+==============
+
+================ ===============================================================
+``DNM``          This PR should not be merged (Do Not Merge).
+                 For work in progress, GitHub "draft" PRs are preferred
+``Stale PR``     PR which seems abandoned, and requires attention by the author
+``Blocked``      Blocked by another PR or issue
+``needs review`` The PR needs attention from the maintainers
+``Backport``     The PR is a backport or should be backported
+``Licensing``    The PR has licensing issues which require a licensing expert to
+                 review it
+================ ===============================================================
+
+Issue only labels
+=================
+
+==================== ===========================================================
+``Regression``       Something, which was working, does not anymore
+                     (bug subtype)
+``question``         This issue is a question to the Zephyr developers
```

Comment:
A consistent first upper-case letter would be good...

Also, are we encouraging folks to ask questions as GitHub issues, or asking them to post questions in Slack?

Reply:
>A consistent first upper-case letter would be good..

Sure

> Also, are we encouraging folks to ask questions as GitHub issues, or asking them to post questions in Slack?

Well that is another thing. The fact is that today, people comes to both places with questions, and some issues are morphed into questions.